eng/pipelines/mgmt-auto-release.yml

```diff
@@ -0,0 +1,54 @@
+trigger: none
+
+pr: none
+
+schedules:
+  - cron: "0 1 * * *"
+    displayName: Daily release
+    branches:
+      include:
+        - main
+    always: true
+
+stages:
+  - stage:
+    jobs:
+      - job:
+        displayName: Go Auto Release
+        steps:
+          - task: Npm@1
+            inputs:
+              command: 'custom'
+              customCommand: 'install -g autorest@latest'
+
+          - task: GoTool@0
+            inputs:
+              version: '1.18.2'
+
+          - script: |
+              export PATH=$PATH:$HOME/go/bin
+              sudo ln -s /usr/bin/pwsh /usr/bin/pwsh.exe
```

Comment:
Why is this necessary? Are there tools in this pipeline that only look for `pwsh.exe`? Can we improve them so they are more cross platform compatible?